docs/continuous-integration/credential-scanning/readme.md

```diff
@@ -1,8 +1,8 @@
-# Credential scanning
```

Comment:
This file should be renamed `docs/continuous-integration/credential-scanning.md`

Reply:
This is a section with recipes so I need to leave it as is unfortunately